grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -278,7 +278,7 @@ class _IncidentPage extends React.Component<IncidentPageProps, IncidentPageState
     const showLinkTo = !incident.dependent_alert_groups.length && !incident.root_alert_group && !incident.resolved;
     const integrationNameWithEmojies = <Emoji text={incident.alert_receive_channel.verbal_name} />;
     const sourceLink = incident?.render_for_web?.source_link;
-    const isServiceNow = incident?.alert_receive_channel?.integration === 'servicenow';
+    const isServiceNow = Boolean(incident?.external_urls?.find((el) => el.integration_type === 'servicenow'));
```

Comment:
`nit` you use "servicenow" string several times, let's maybe extract this string to the const like `INTEGRATION_SERVICENOW='servicenow'` or something

Reply:
Done